odb_extractor_atoms.py

```diff
@@ -0,0 +1,627 @@
+#!/usr/bin/env python
```

Comment:
For example scripts like this, maybe we should put them in their own package for reference and to not muddy the waters with the actual Scheduler code.

Reply:
That would be fine. Maybe in my old 'greedy_max' repo?